SDNExpress/scripts/SDNExpress.ps1

```diff
@@ -2401,6 +2401,215 @@ Configuration ConfigureHostAgent
     }
 }
 
+Configuration ConfigureIDns
+{
+    
+	Import-DscResource -ModuleName 'PSDesiredStateConfiguration'
+
+	Node $AllNodes.Where{$_.Role -eq "NetworkController"}.NodeName
+

```

Comment:
There are a bunch of blank lines and alignment issues in this Configuration. Can you please remove the blank lines and ensure that you are using spaces (not tabs) for indentation. This is for consistency with the rest of the .ps1 file.